src/zookeeper/build/run.sh

```diff
@@ -20,9 +20,19 @@
 sed -i "/^JAVA_OPTS.*/ s:.*:JVMFLAGS=\"${JAVA_OPTS}\":" /usr/share/zookeeper/bin/zkEnv.sh
 
 HOST_NAME=`hostname`
-/usr/local/host-configure.py -c /host-configuration/host-configuration.yaml -f /etc/zookeeper/conf/zoo.cfg -n $HOST_NAME
-cp /myid /var/lib/zoodata/myid
-/usr/local/host-configure.py -c /host-configuration/host-configuration.yaml -f /var/lib/zoodata/myid -n $HOST_NAME
+
+# create /etc/zookeeper/conf/zoo.cfg
+echo "
+tickTime=2000
+dataDir=/var/lib/zoodata
+clientPort=2181
+initLimit=5
+syncLimit=2
+${ZK_SERVERS}
+" > /etc/zookeeper/conf/zoo.cfg
+
+# generate a zookeeper server id from ip: remove the dot, and reserve the last 7 digits.
+zkid=${POD_IP//\./} && echo ${zkid: -7} > /var/lib/zoodata/myid
```

Comment:
Why not just use the Ip digit without dot? I think that value will be unique.

Reply:
The assumption based on the rule of ip range.
Can't image zookeeper working crossing such a huge network.

Not using full 9 digital of ipv4 is that it may overflow integer.

Reply:
> The assumption based on the rule of ip range.
> Can't image zookeeper working crossing such a huge network.
> 
> Not using full 9 digital of ipv4 is that it may overflow integer.

Even you use full 9 digital, the problem still exits. Can we find another solution?

Reply:
Well, what about using sha1sum?
It will be better, but won't guarantee the unique of server id.

Reply:
Little confused. I still remember sha1sum is your first solution. Why did you move to the solution of the last 7 digit of ip address.

Reply:
When updating the doc, I realized that I was generating the wrong format `ZK_SERVERS`, the`zkid` was missing.
So I have to generate the zkid in `zookeeper.py` and `bash`, but I didn't want to introduce more dependency into the python script.